transcoder/source/file/LineDelimitedFileMessageSource.py

```diff
@@ -49,3 +51,18 @@ def get_message_iterator(self):
             self.increment_count()
             yield self.decode_message(line)
             self._log_percentage_read()
+
+    def decode_message(self, record):
```

Comment:
I think this should stay at the FileMessageSource level, no? There may be other file record level sources that can use it too.